src/violin.jl

```diff
@@ -1,11 +1,22 @@
-
 # ---------------------------------------------------------------------------
-# Violin Plot
+# Violin plot utility functions
 
 const _violin_warned = [false]
 
-function violin_coords(y; trim::Bool=false)
-    kd = KernelDensity.kde(y, npoints = 200)
+"""
+**Use kde to return an enveloppe for the violin and beeswarm plots**
+
+~~~
```

Comment:
Thanks for adding a good docstring - this is missing from too many places in StatPlots. For consistency, could you use the format used in `groupederror.jl` (indent call 4 spaces, no `~~~`)? It's fine to keep the  argument specifications.